common/config/azure-pipelines/npm-publish-tarballs.yaml

```diff
@@ -0,0 +1,59 @@
+parameters:
+  - name: TestRun
+    displayName: 'Test Run (Will Not Create Tar Files for Rushstack)'
```

Comment:
Considering that publishing the artifact does not have any direct consequences, there's no reason to have a dryRun option for this pipeline.